third_party/txt/tests/txt_test_utils.cc

```diff
@@ -16,6 +16,8 @@
 
 #include "txt_test_utils.h"
 
+#include <sstream>
```

Comment:
Erroneous include?

Reply:
Through a dependency chain, this was somehow being included earlier (probably `iosfwd` in the old string view). `std::stringstream` is used below so this is as close to IWYU as I could get.